Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:latest AS golang
+WORKDIR /go/src/github.com/bitly/oauth2_proxy
+RUN go get -u github.com/golang/dep/cmd/dep
+COPY . .
+#RUN dep ensure && \
+RUN go get ./...
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o oauth2_proxy .
+
+FROM scratch
```

Comment:
Does the Dockerfile accept multiple FROM statements?

Reply:
His Dockerfile adopts the multi-stage feature added recently to Docker: https://docs.docker.com/develop/develop-images/multistage-build/#before-multi-stage-builds

Reply:
Ack.